### DISCUSSION TITLE
zh/second_lang/Japanese/entry_lesson_02

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#460] [Announcements]
# zh/second_lang/Japanese/entry_lesson_02

代词 これ/それ/あれは～です これ/それ/あれ/どれ：(事物性指示代词，一般用于指代物或事)这个/那个/那个/哪个。 - これ 距离说话人近，听话人远的物体。 - それ 距离说话人远，听话人近的物体。 - あれ 距离说话人和听话人都远的`物体。 - どれ 询问哪个? 例句 だれですか。/何ですか。 出现针对性的特殊疑问词的疑问句叫做特殊疑问句，句末仍要用「か」来表示疑问。回答针对特殊疑问词作出相应具体回答即可。 例1：これは何ですか。——それは本です。

https://notes.solex-inc.com/zh/second_lang/Japanese/entry_lesson_02.html

**Comment by 100vision** [comment]
测试
